src/com/company/FileUtils.java

```diff
@@ -2,67 +2,49 @@
 
 import java.io.File;
 import java.io.FileWriter;
-import java.io.IOException;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Scanner;
-import static com.company.Main.*;
-
 
 public class FileUtils {
-    // method to give name for file where sorted numbers will be written(if that file name is empty)
-    public static boolean isFileExists(){
-
-        boolean fileExist = false;
 
-        try{
-            String d = fileToSave.toString();
-            if(d.equals(".txt")) {
-                fileToSave = new File("./results/sortedNumbers.txt");//
-            }
-            fileExist = true;
+    public static List<Integer> unsortedNumbers;
```

Comment:
remove this

Reply:
" public static List<Integer> unsortedNumbers;" is removed